analysed-packages/dbus/version-1.14.6/dbus-1.14.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,13415 @@
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/build-aux/install-sh
+SPDXID: SPDXRef-item546371
+FileChecksum: SHA1: 894e0a9de7976233743575c6b84dedc1b37025b6
+FileChecksum: SHA256: 3d7488bebd0cfc9b5c440c55d5b44f1c6e2e3d3e19894821bae4a27f9307f1d2
+FileChecksum: MD5: b090c5891571c820e91d345a9551af44
+LicenseConcluded: LicenseRef-Public-domain AND LicenseRef-X11
+LicenseComments: <text> Additionally to the X11 license, the text in this file is: "FSF changes to this file are in the public domain." Thus we conlcuded additionally public domain </text>
+LicenseInfoInFile: LicenseRef-X11
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-Public-domain
+FileCopyrightText: <text> Copyright (C) 1994 X Consortium </text>
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/build-aux/ltmain.sh
+SPDXID: SPDXRef-item546370
+FileChecksum: SHA1: fb820158aaf72017a37b29d0ebcd4ad8e98268b8
+FileChecksum: SHA256: 73bc7975f9b40d09b6482514b7db0d6950a6230d9d0b2ca6af9640488a98c7c6
+FileChecksum: MD5: f93f8aa2e516d29f00960b4c5e41cbcd
+LicenseConcluded:  (LicenseRef-GPL-2.0-or-later OR LicenseRef-MIT) AND LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception AND LicenseRef-warranty-disclaimer-da410606429db1ac8820308d4a486923
+LicenseComments: <text> Additionally to the GPL-2.0-or-later with Libtool-exception and warranty-disclaimer boilerplates there is the following information in the file:
+
+ This file is dual licensed under the terms of the MIT license
+# <https://opensource.org/license/MIT>, and GPL version 2 or later
+# <http://www.gnu.org/licenses/gpl-2.0.html>. You must apply one of
+# these licenses when using or redistributing this software or any of
+# the files within it. See the URLs above, or the file `LICENSE`
+# included in the Bootstrap distribution for the full license texts. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-Dual-license
+LicenseInfoInFile: LicenseRef-Libtool-exception
+FileCopyrightText: <text> Copyright (C) 1996-2019, 2021-2022 Free Software Foundation, Inc.
+Copyright (C) 2004-2019, 2021 Bootstrap Authors
+Copyright (C) 2010-2019, 2021 Bootstrap Authors </text>
+FileAttributionText: <text> ltmain.sh: To the extent that files may be licensed under either GPL-2.0-or-later or MIT, in this context, MIT has been chosen. This choice shall not restrict the freedom of other users to select either GPL-2.0-or-later  or MIT licenses. For convenience, all license texts are provided. </text>
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/bus/main.c
+SPDXID: SPDXRef-item546316
+FileChecksum: SHA1: 4646c52a0dedd140d52a279474e1cb35d08cf5b2
+FileChecksum: SHA256: 2b375c24cac0e7c452cd386d4c0561888478e2774d335d765ba6e1e68a284812
+FileChecksum: MD5: c8e595fb31ccc9cab0e28760aef13f15
+LicenseConcluded: (LicenseRef-AFL-2.1 OR LicenseRef-GPL-2.0-or-later) AND LicenseRef-warranty-disclaimer-ddcccd98b144520c9d6a4fc3fe1998d6
```

Comment:
Legal has told us to keep all the licenses that are included in variables. The reason behind that was in case you are not 100% sure that this variable is going to be used in your final software that you deliver to customer. then it should be present there. So she mentioned "please provide always more information rather than less".

So even if you are sure 99% that is not used in delivered software but 1% unsure. Please include those licenses. In the worst hypothesis, you have provided more than necessary and that is still ok.